__tests__/app.test.js

```diff
@@ -55,4 +54,41 @@ describe("General errors", ()=>{
       expect(body).toEqual({})
     })
   })
+})
+
+describe("GET /api/articles/:article_id", ()=>{
+  test("200: responds with an article object with correct properties", ()=>{
+    return request(app)
+    .get('/api/articles/1')
+    .expect(200)
+    .then(({body:{article}})=>{
+      console.log(article)
+        expect(article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        })
+    })
+  })
+  test("404: responds with 'Resource not found' if there are no matches for article ID", ()=>{
+    return request(app)
+    .get('/api/articles/100')
+    .expect(404)
+    .then(({text})=>{
+     
+      expect(text).toBe('Resource not found')
+    })
+  })
+  test.only("404: responds with 'Not Found' if ID is missing", ()=>{
+    return request(app)
+    .get('/api/articles/')
+    .expect(404)
+    //I don't know how to handle 404 when it's provided by express so yeah
```

Comment:
try adding a `.then` block with a parameter, then console log the outcome. See what you get!